lib/honeybadger/plugins/active_job.rb

```diff
@@ -20,7 +20,7 @@ def perform_around(job, block)
 
         def context(job) # rubocop:disable Metrics/MethodLength
           {
-            component: job.class,
+            component: 'active_job',
```

Comment:
This may be OK, considering that `job.class` is captured with `job_class` below, but I'll let a ruby dev approve :).